src/storekit.cs

```diff
@@ -852,13 +888,17 @@ interface SKStorefront {
 
 	interface ISKPaymentQueueDelegate {}
 
-	[Mac (10,15), iOS (13,0)]
+	[Watch (6,2), Mac (10,15), iOS (13,0)]
 	[Protocol]
 	[Model (AutoGeneratedName = true)]
 	[BaseType (typeof(NSObject))]
 	interface SKPaymentQueueDelegate {
 		[Export ("paymentQueue:shouldContinueTransaction:inStorefront:")]
 		bool ShouldContinueTransaction (SKPaymentQueue paymentQueue, SKPaymentTransaction transaction, SKStorefront newStorefront);
+
+		[NoWatch, NoMac, iOS (13,4)]
+		[Export ("paymentQueueShouldShowPriceConsent:")]
+		bool PaymentQueueShouldShowPriceConsent (SKPaymentQueue paymentQueue);
```

Comment:
`PaymentQueue` prefix comes from the protocol and the fact that there's only one parameter
It's not needed (like the API above)